src/app/pages/AccountPage/Features/SendTransaction/index.tsx

```diff
@@ -25,7 +25,7 @@ export function SendTransaction(props: SendTransactionProps) {
   const { error, success } = useSelector(selectTransaction)
   const validators = useSelector(selectValidators)
   const [recipient, setRecipient] = useState('')
-  const [amount, setAmount] = useState('')
+  const [amount, setAmount] = useState('0')
```

Comment:
We really don't want to show a placeholder to user?

Reply:
The idea was that the amount field should be initialized in a valid state, which can be submitted right away.
If we show the "0" as a placeholder (as before), it gives the false impression that the field is valid, when in fact it is not.

With this change, we actually put 0 into the field as the initial value, so it's valid.

And if it's removed, then we show the real placeholder, see below.

> placeholder={t('account.sendTransaction.enterAmount', 'Enter an amount')}

Reply:
> I would validate this with Don.

Discussed with Don; we will keep the empty starting value (but replace the placeholder.)